website/docs/components/time/partials/code/component-api.md

```diff
@@ -0,0 +1,60 @@
+## Component API
+
+The Time component makes use of the Hds-time service, which provides various time and formatting related functions as well as a tracked `Now` property representing current time in milliseconds, which continuously updates.
+
+It uses two helper functions: the format-date helper, which formats a date according to the passed in options, and the format-relative helper, which returns a relative string according to the passed in value and units.
+
+### Time component
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="date" @type="Date | string">
+    Specify the date and time for a single date. Value can be specified as a JavaScript Date object or as an ISO date string.
```

Comment:
suggestion: if not too much trouble, we can link to the MDN spec for people less familiar with the object
```suggestion
    Specify the date and time for a single date. Value can be specified as a [JavaScript Date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) or as an ISO date string.
```